tests/e2e/Pipfile

```diff
@@ -0,0 +1,22 @@
+[[source]]
+
+url = "https://pypi.python.org/simple"
+verify_ssl = true
+name = "pypi"
+
+
+[dev-packages]
+
+
+
+[packages]
+
+"flake8" = "==3.5.0"
+"flake8-isort" = "==2.4"
+mozlog = "==3.7"
+pytest = "==3.4.2"
+pytest-base-url = "==1.4.1"
+pytest-metadata = "==1.6.0"
```

Comment:
This is a dependency of base-url and html, so we could probably remove it from here.

Reply:
Fixed, and re-built.